app/models/record.rb

```diff
@@ -8,13 +8,14 @@
 # the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
+# Record
 class Record
   include Supplejack::Record
 
-  def image_url(options={})
+  def image_url(options = {})
     size = '204'
     size = "#{options[:width]}" unless options == {}
-    source_url = self.thumbnail_url || ""
+    source_url = thumbnail_url || ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.